auto_dev/data/templates/customs/method_template.jinja

```diff
@@ -1,49 +1,51 @@
-    def {{ method_name }}(self, message: ApiHttpMessage{% for param in path_params_snake_case %}, {{ param }}{% endfor %}{% if method|lower in ['post', 'put', 'patch', 'delete'] %}, body{% endif %}):
+    {%- if use_daos %}
```

Comment:
Where are the imports here?
Also need the copyright header no? 
Otherwise no go on linting right?

Reply:
imports and copyright is handled by https://github.com/8ball030/auto_dev/blob/feat/openapi-augment-fixes/auto_dev/data/templates/customs/handler_header.jinja